lte/gateway/c/session_manager/test/test_usage_monitor.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <chrono>
+#include <thread>
+
+#include "SessionState.h"
+#include "SessiondMocks.h"
+#include "ProtobufCreators.h"
+#include "SessionStateTester.h"
+
+#include <gtest/gtest.h>
+
+using ::testing::Test;
+
+namespace magma {
+TEST_F(SessionStateTest, test_insert_monitor) {
+  EXPECT_EQ(update_criteria.static_rules_to_install.size(), 0);
```

Comment:
maybe it is evident, but I don't see it right now. Where does `update_criteria` object comes from?

Reply:
`TEST_F(SessionStateTest, test_insert_monitor)` indicates that it is using the testing class `SessionStateTest`, newly moved to `SessionStateTester.h`. So it is defined there. I think it's probably better in this test to instantiate its own update_criteria though. Will change that.

Reply:
ohh great, i missed to check the signature 👍